app/services/foreman/renderer/scope/macros/helpers.rb

```diff
@@ -181,6 +181,21 @@ def generate_web_request(utility:, url:, ssl_ca_cert: nil, headers: nil, params:
             utility[:format_params].call(params).each { |param| command << param } if params
             command.join(" \\\n  ")
           end
+
+          apipie :method, 'Generates shell command to restart the given service' do
+            required :service, String, desc: 'Name of the service to restart'
+            returns String, desc: 'Shell command to restart the given service'
+            example 'service_restart("sshd") #=> "if command -v systemctl >/dev/null 2>&1; then\n  systemctl restart sshd\nelse\n  service sshd restart\nfi"'
+          end
+          def service_restart(service)
+            <<-SHELL.strip
+              if command -v systemctl >/dev/null 2>&1; then
+                systemctl restart #{service}
+              else
+                service #{service} restart
```

Comment:
Is non-systemd still something we need to care about? Does ssh on EL6 even support SSH certifcates?